apps/nextra/pages/en/network/blockchain/execution.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Parallel Execution"
+---
+
+# Execution
+
+In blockchains, execution refers to the process by which validators 
+run and apply smart contract transactions from ordered blocks to update the blockchain’s state.
+
+It is crucial because validators must agree on the same final state, post-execution,
+ensuring consistency and security across the network.
+
+Efficient execution, particularly parallel execution, is key to scaling blockchain performance and throughput.
+
+## Why Parallel Execution?
+
+![](https://miro.medium.com/v2/resize:fit:4800/format:webp/0*qG0KRe1KzsP8Odth)
+
+Smart contract execution is a major throughput bottleneck for blockchains. 
+After proposing blocks and agreeing on their order, validators have to execute the 
+transactions in the ordered blocks. Crucially, validators must arrive at the same final state, 
```

Comment:
Agreeing on order is the order of the transactions not the blocks.

Reply:
For context, I had a discussion with @sasha8 and @bchocho on how this page should be structured and we decided to use the text originally included in the Block-STM blog, so this is copied verbatim from https://medium.com/aptoslabs/block-stm-how-we-execute-over-160k-transactions-per-second-on-the-aptos-blockchain-3b003657e4ba. 

Will let the author(s) get first-pick on deciding the language here and for the other suggestions here cc @sasha8. If the wording is incorrect, we should maybe consider updating it in the blog as well.

Reply:
it is a nit. I have no strong opinion.